docs/advanced-features/tabulator.md

```diff
@@ -0,0 +1,100 @@
+Here is the revised version with the API call information moved into the **Overview** section, and the **Configuration** section focused solely on the YAML file:
+
+---
+
+# Tabulator Configuration for Quilt
+
+> This feature requires Quilt stack version 1.55.0 or higher
+
+## Overview
+
+Tabulator allows Quilt admins to configure tables for querying tabular data
+objects across multiple packages using AWS Athena. Admins can define schemas
+and data sources for CSV, TSV, Parquet, and other formats, enabling users to
+run SQL queries directly on Quilt packages.
+
+The configuration can be written in YAML and managed using the
+`quilt3.admin.tabulator_config.set()` function or via the Quilt Admin UI.
+
+The API call for setting up the Tabulator configuration accepts three parameters:
+
+1. `bucket_name: str`: The name of the S3 bucket to be searched, e.g., "udp-spec".
+2. `table_name: str`: The name of the table to be created, e.g., "ccle-tsv".
+3. `config: str`: The configuration for the table as a YAML string.
```

Comment:
that looks like duplication of info from API reference

Reply:
Ah, forgot about that. remove.